mappings/59e94d6c94fe69adea7a424688d8fea9877dc4ca6b33865e98915337476f6f6d626c65.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "59e94d6c94fe69adea7a424688d8fea9877dc4ca6b33865e98915337476f6f6d626c65",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://twitter.com/ReviewOkinawa",
```

Comment:
Hi @UYMYsi,

can you please clarify how this token relates to the twitter account in the URL value?